irohad/ordering/impl/on_demand_ordering_service_impl.hpp

```diff
@@ -9,14 +9,13 @@
 #include "ordering/on_demand_ordering_service.hpp"
 
 #include <map>
-#include <queue>
 #include <shared_mutex>
-#include <unordered_map>
 
 #include <tbb/concurrent_unordered_set.h>
 #include "interfaces/iroha_internal/unsafe_proposal_factory.hpp"
 #include "logger/logger_fwd.hpp"
 #include "multi_sig_transactions/hash.hpp"
+#include "multi_sig_transactions/state/mst_state.hpp"
```

Comment:
I think better to leave a todo about changing a place of PointerBatchHasher and BatchHashEquality